dex/networks/btc/script.go

```diff
@@ -170,6 +170,15 @@ const (
 	// NOTE: witness data is not script.
 	RedeemP2WPKHInputWitnessWeight = 1 + 1 + DERSigLength + 1 + 33 // 109
 
+	// RedeemP2PKHInputTotalSize is the worst case size of a transaction
+	// input redeeming a P2WPKH output and the corresponding witness data.
+	// It is calculated as:
+	//
+	// 41 vbytes base tx input
+	// 109wu witness +  2wu segwit marker and flag = 28 vbytes
+	// total = 69 vbytes
+	RedeemP2PWKHInputTotalSize = RedeemP2WPKHInputSize + ((RedeemP2WPKHInputWitnessWeight + 2 + 3) / 4)
+
```

Comment:
I'm so confused about wu/vbyte/byte

Is a vbyte just a byte? And google tells me 4wu is one byte? So the `2wu` there is half a byte?

Reply:
One byte of regular data is 4wu, but 1 byte of witness data is just 4wu. Since this is part of the witness data, 2wu = 2 bytes.

Reply:
Ah yeah, I meant 1 byte of witness data is 1wu.